### PR TITLE
8277361: java/nio/channels/Channels/ReadXBytes.java fails with OOM error

### PR DESCRIPTION
**Pull request by bplb**
This request would increase the amount of heap memory that the test can use, and decrease execution time by not writing the entirety of test files.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8277361](https://bugs.openjdk.java.net/browse/JDK-8277361): java/nio/channels/Channels/ReadXBytes.java   fails with OOM error

### Reviewers
 * [Alan Bateman](https://openjdk.java.net/census#alanb) (@AlanBateman - **Reviewer**) ⚠️ Review applies to f606ce8cd32dd805c06ad6e2ff4399deff63f7d2
 * [Lance Andersen](https://openjdk.java.net/census#lancea) (@LanceAndersen - **Reviewer**) ⚠️ Review applies to f606ce8cd32dd805c06ad6e2ff4399deff63f7d2

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/6732/head:pull/6732` \
`$ git checkout pull/6732`

Update a local copy of the PR: \
`$ git checkout pull/6732` \
`$ git pull https://git.openjdk.java.net/jdk pull/6732/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 6732`

View PR using the GUI difftool: \
`$ git pr show -t 6732`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/6732.diff">https://git.openjdk.java.net/jdk/pull/6732.diff</a>

</details>

### CODE REVIEW

**Comment by bplb** [comment]
The execution time of the test appeared to be decreased around 25% by writing fewer bytes to test files.

File: test/jdk/java/nio/channels/Channels/ReadXBytes.java [hunk 1]
```diff
@@ -72,30 +73,50 @@
...
+
     // Creates a temporary file of a specified length with undefined content
     static Path createFile(long length) throws IOException {
-        File file = File.createTempFile("foo", ".bar");
-        file.deleteOnExit();
-        try (RandomAccessFile raf = new RandomAccessFile(file, "rw")) {
-            raf.setLength(length);
+        Path path = createFilePath();
+        path.toFile().deleteOnExit();
+        try (FileChannel fc = FileChannel.open(path, CREATE_NEW, SPARSE, WRITE);) {
```
Comment (AlanBateman) [inline test/jdk/java/nio/channels/Channels/ReadXBytes.java:91]: Spurious semicolon at the end.

**Review by AlanBateman — APPROVED** [review]

**Review by LanceAndersen — APPROVED** [review]
Looks good Brian

Minor suggestion below

**Pull request closed without merging**